### PR TITLE
fix(private-network): amend guide

### PR DESCRIPTION
**Pull request by BrunoCampana**
The goal of this PR is update the guide about how to install a private testnet.

The current version uses CPU miner to mine blocks.

The new version uses Transaction mining services to do this.

- Section about CPU miner was removed.
- Diagram was updated.
- All references about CPU miner were removed.
- The explanation about tx mining service mining blocks was added.

### CODE REVIEW

File: text/0033-private-network-guide.md [hunk 1]
```diff
@@ -7,19 +7,17 @@
...
-- One cpu-miner mining blocks in the network
-- One tx-mining-service to be used by Wallets to mine transactions
+- One tx-mining-service to be used by wallets to mine transactions and used by the full nodes to mine blocks
 - One instance of hathor-wallet-headless
 
 The diagram below shows the overall architecture of this private network:
 
-![fig-architecture](./0033-images/private-network-architecture.png "Private network architecture")
-
+![fig-architecture](./0033-images/private-network-architecture-2.png "Private network architecture")
```
Comment (luislhl) [inline text/0033-private-network-guide.md:20]: Should this image show the CPU miner connected to the tx-mining-service?

File: text/0033-private-network-guide.md [hunk 2]
```diff
@@ -7,19 +7,17 @@
...
 
-This guide will show you how to run a local private Hathor network with its own full nodes, miner, wallet and explorer. Note that a private network will be completely separated from Hathor's mainnet and testnet.
+This guide will show you how to run a local private Hathor network with its own full nodes, tx mining service, wallet and explorer. Note that a private network will be completely separated from Hathor's mainnet and testnet.
 
 By the end of the guide, you will have a fully working private network with the following setup:
 
 - Three hathor-core full nodes connected between each other
-- One cpu-miner mining blocks in the network
-- One tx-mining-service to be used by Wallets to mine transactions
+- One tx-mining-service to be used by wallets to mine transactions and used by the full nodes to mine blocks
```
Comment (luislhl) [inline text/0033-private-network-guide.md:15]:
```suggestion
- One tx-mining-service to be used by Wallets to mine transactions
- One cpu-miner connected to the tx-mining-service to mine blocks
```

What do you think?

File: text/0033-private-network-guide.md [hunk 3]
```diff
@@ -52,18 +50,15 @@ Here is a table summarizing the minimum requirements for each type of instance:
 |---  | ---:  | ---: | ---: | ---:     | --- |
 |Full node         | 3 | 4 | 8 GB | 4 GB | hathornetwork/hathor-core
 |Wallet            | 1 | 2 | 2 GB | 100 MB | hathornetwork/hathor-wallet-headless
-|Miner             | 2 | 2 | 1 GB | 100 MB | hathornetwork/cpuminer
```
Comment (luislhl) [inline text/0033-private-network-guide.md:55]: We will still have a miner connected to the tx-mining-service. Maybe we should keep this line.

File: text/0033-private-network-guide.md [hunk 4]
```diff
@@ -52,18 +50,15 @@ Here is a table summarizing the minimum requirements for each type of instance:
...
-
 # Creating a new network
 
-The following steps must be executed to create a new network:
+The following steps should be executed to create a new network:
 
 1. Generate the configuration of the new network.
 2. Launch the full nodes and build the new p2p network.
 3. Launch the tx-mining-service to be able to send transactions.
-4. Launch the miner connected to the tx-mining-service.
```
Comment (luislhl) [inline text/0033-private-network-guide.md:66]: This step still exists in the text

File: text/0033-private-network-guide.md (hunk 2 above)
Comment (BrunoCampana) [reply text/0033-private-network-guide.md:15]: OK: change made:
I did not directly commit your suggestion, but added it in my commit.

File: text/0033-private-network-guide.md (hunk 3 above)
Comment (BrunoCampana) [reply text/0033-private-network-guide.md:55]: OK: change made: 
Readded the line as suggested.

File: text/0033-private-network-guide.md [hunk 5]
```diff
@@ -42,7 +40,7 @@ For full nodes, we recommend a minimum of 4 CPUs, 8 GB of RAM, and 4 GB of stora
 
 For wallets, we recommend a minimum of 2 CPUs, 2 GB of RAM, and 100 MB of storage. Wallets are usually memory bounded.
 
-For miners, we recommend a minimum of 2 CPUs and 1 GB of RAM, and 100 MB of storage. CPU miners are usually CPU bounded.
+For tx mining service, we recommend a minimum of 2 CPUs and 1 GB of RAM, and 100 MB of storage. CPU miners are usually CPU bounded.
```
[the remark this answers is not in the document]
Comment (BrunoCampana) [reply text/0033-private-network-guide.md:43]: OK: change made:
Readded the line as suggested.

File: text/0033-private-network-guide.md (hunk 4 above)
Comment (BrunoCampana) [reply text/0033-private-network-guide.md:66]: OK: change made:
Readded the step as suggested.

File: text/0033-private-network-guide.md (hunk 1 above)
Comment (BrunoCampana) [reply text/0033-private-network-guide.md:22]: OK: change made:
The diagram was updated as suggested, readding the cpu miner, but now connected to and only to tx-mining-service.

File: text/0033-private-network-guide.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (luislhl) [reply text/0033-private-network-guide.md:45]: The sentence `CPU miners are usually CPU bounded.` at the end is not making sense.

[the remark this answers is not in the document]
Comment (BrunoCampana) [reply text/0033-private-network-guide.md:45]: OK: change made:
Removed sentence.

**Review by luislhl — APPROVED** [review]

**Review by jansegre — APPROVED** [review]

**Pull request merged**